packages/admin/admin-rte/src/core/translation/stateToHtml.spec.ts

```diff
@@ -0,0 +1,341 @@
+import { convertFromRaw, EditorState, RawDraftContentState } from "draft-js";
+
+import { IRteOptions } from "../Rte";
+import { stateToHtml } from "./stateToHtml";
+
+// Remove all newlines and spaces to compare the html strings
+function trimHtml(html: string) {
+    return html.replace(/\s|\n/g, "");
+}
```

Comment:
I'd suggest we'd do this directly in stateToHtml after transformation, what do you think?

Reply:
Would need to adjust the trim function since i remove all spaces for the test compare. And would then have to check if it works fine for the translation replacing all sequences of two or more spaces with just one space and check the newlines.

Don't know if that is worth it.

Reply:
Now that I think of it, I don't believe that's possible at all, as there could be whitespaces and line breaks in text, which we shouldn't remove. Odd that [draft-js-export-html](https://www.npmjs.com/package/draft-js-export-html) has no option to not add indentation and line breaks 🤷🏼‍♂️ 

Let's leave it like this.